tests/l2/test_l2_configure.py

```diff
@@ -0,0 +1,91 @@
+"""
+Tests related to L2 configuration
+"""
+import logging
+import pytest
+import time
+
+from tests.common import reboot
+from tests.common.helpers.assertions import pytest_expect
+
+logger = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.topology('any')
+]
+
+
+def test_l2_configure(duthosts, rand_one_dut_hostname):
+    """
+    @summary: Test configuring dut as a L2 switch.
+
+    Args:
+        duthosts: set of DUTs.
+    """
+    # Setup.
```

Comment:
> # Setup [](http://example.com/codeflow?start=3&length=8)

try this way to setup

[https://github.com/sonic-net/sonic-mgmt/blob/master/tests/generic_config_updater/test_aaa.py#L33](https://github.com/sonic-net/sonic-mgmt/blob/master/tests/generic_config_updater/test_aaa.py#L33)